### ISSUE TITLE
Add OpenStreetMap_Places as a search table

### ISSUE THREAD

**Issue by stdavis** [agrc/masquerade#43]
I'm guessing that it will replace some existing tables as well.

Ref: https://github.com/agrc/porter/issues/180

**Comment by steveoh** [comment]
My understanding is that it is supplemental and will not be replacing anything in the SGID. Unsure if it will replace anything in masquerade.

**Comment by stdavis** [comment]
Yes, I meant that it will replace some or [these tables](https://github.com/agrc/masquerade/blob/main/src/masquerade/providers/open_sgid.py#L290-L332).

**Comment by stdavis** [comment]
We also may want to add the idea of context when we add this provider. For example, if you are searching for "Starbucks" it would be helpful to have the city.

**Comment by steveoh** [comment]
This would "fix" the daybreak query we chatted about in #data
